chancacher/chancacher_test.go

```diff
@@ -697,6 +703,136 @@ func TestCacheMaxSize(t *testing.T) {
 	}
 }
 
+// TestCacheEntries verifies that we can write entries, with EVs
+// attached, and read them back out.
+func TestCacheEntries(t *testing.T) {
+	gob.Register(&entry.Entry{})
```

Comment:
if you have to register them in this test we probably need to register them in the main chancacher package.